Parser/asdl_c.py

```diff
@@ -1176,18 +1187,19 @@ class PartingShots(StaticVisitor):
 }
 
 /* mode is 0 for "exec", 1 for "eval" and 2 for "single" input */
+/* and 3 for "func_type" */
 mod_ty PyAST_obj2mod(PyObject* ast, PyArena* arena, int mode)
 {
     mod_ty res;
     PyObject *req_type[3];
-    char *req_name[] = {"Module", "Expression", "Interactive"};
+    char *req_name[] = {"Module", "Expression", "Interactive", "FunctionType"};
     int isinstance;
 
     req_type[0] = (PyObject*)Module_type;
     req_type[1] = (PyObject*)Expression_type;
     req_type[2] = (PyObject*)Interactive_type;
```

Comment:
Shouldn't we be also populating `req_type[3]` here to `FunctionType_type`? The assymetry looks brittle to me.

Reply:
Hm... I misunderstood what this is for. Fortunately the code in builtin_compile_impl() prevents this to be reached with mode==3. I propose to just drop support for FunctionType here.